### PR TITLE
Use JitStdOutFile for .dasm file creation

### PR DESCRIPTION
**Pull request by kunalspathak**
We can use `COMPlus_JitStdOutFile` variable to create `.dasm` files which is slightly faster instead of using `stdout`.

### CODE REVIEW

**Comment by kunalspathak** [comment]
@dotnet/jit-contrib @BruceForstall

File: src/coreclr/scripts/superpmi.py [hunk 1]
```diff
@@ -1634,11 +1634,10 @@ async def create_replay_artifacts(print_prefix, item, self, mch_file, env, jit_d
...
                                 command = [self.superpmi_path] + flags + [jit_path, mch_file]
                                 item_path = os.path.join(location, "{}{}".format(item, extension))
-                                with open(item_path, 'w') as file_handle:
-                                    logging.debug("%sGenerating %s", print_prefix, item_path)
-                                    logging.debug("%sInvoking: %s", print_prefix, " ".join(command))
-                                    proc = await asyncio.create_subprocess_shell(" ".join(command), stdout=file_handle, stderr=asyncio.subprocess.PIPE, env=env)
-                                    await proc.communicate()
+                                modified_env = env.copy()
+                                modified_env['COMPlus_JitStdOutFile'] = item_path
+                                proc = await asyncio.create_subprocess_shell(" ".join(command), stderr=asyncio.subprocess.PIPE, env=modified_env)
```
Comment (BruceForstall) [inline src/coreclr/scripts/superpmi.py:1639]: Why remove the two `logging.debug` commands? Those look useful to me.

Comment (kunalspathak) [reply src/coreclr/scripts/superpmi.py:1641]: Yes, that was by mistake. Put it back.

**Review by BruceForstall — APPROVED** [review]
LGTM. You could manually trigger the superpmi-diffs pipeline on this to make sure it works there.

**Pull request merged**